src/Domain/SalesforceProxy.php

```diff
@@ -33,4 +34,20 @@ public function setSalesforceId(string $salesforceId): void
     {
         $this->salesforceId = $salesforceId;
     }
+
+    /**
+     * @return DateTime
+     */
+    public function getCreatedAt(): ?DateTime
```

Comment:
These are both still wrong for the same reasons! The getters need to be defined at the same level as what they read from so that they are guaranteed to work.

Reply:
@NoelLH i'm not quite sure what you mean, where should they live?